test/modern_bpf/event_class/event_class.cpp

```diff
@@ -166,6 +147,8 @@ event_test::event_test(int syscall_id, int event_direction)
 event_test::event_test()
 {
 	m_current_param = 0;
+	m_event_type = PPM_EVENT_MAX;
```

Comment:
Canary value to let `enable_capture` know it must attach both sys_enter and sys_exit tracepoints.